### ISSUE TITLE
Using "norom" with different arch (spc700 as the test case) causes Unknown Command error

### ISSUE THREAD

**Issue by KungFuFurby** [RPGHacker/asar#208]
```
arch spc700
norom
;code here
```
without any other architecture specified prior causes this error (example from testing this out in AddMusicKFF):
```
asm/main.asm:170: error: (E5117): Unknown command. [norom]
```

Workaround is to swap the two and add `arch 65816` on top for now, like this...
```
arch 65816
norom
arch spc700
;code here
```

**Comment by p4plus2** [comment]
Fixed.

**Comment by KungFuFurby** [comment]
Noted as closed with commit d1419d4a943ca92c629fad471b424d0b78f88cf8.
